applications/crossbar/priv/couchdb/schemas/system_config.callflow.json

```diff
@@ -150,9 +150,13 @@
                     "description": "callflow save after notify",
                     "type": "boolean"
                 },
-                "vm_message_foraward_type": {
+                "vm_message_forward_type": {
                     "default": "only_forward",
-                    "description": "Enable or disable the ability to prepend a message when forwarding a voicemail message",
+                    "description": "callflow voicemail vm_message_forward_type",
```

Comment:
Why changing the description?

Reply:
Oops, I didn't realize the old one had a "proper" description :)